### ISSUE TITLE
Clicking Create Server more than once creates multiple interface entries

### ISSUE THREAD

**Issue by bmcgonag** [bmcgonag/Vector#13]
For the "Server" interface creation, if I have an error, I have to click the button again, but this causes there to be multiple "[Interface]" sections in the server configuration file.

Fix:
don't let there be more than 1 "[Interface]" section created.

**Comment by bodzio4749** [comment]
I don't believe it is fixed. I download the installation script last night and still getting multiple entries for the same default value.  Does "manual" overwrite work, or still "under construction"?
The configuration values from /etc/wireshark are ignored, and actually overwritten. 
As I was finishing to play, I noticed that digitalocean droplet status was showing 100% CPU utilization, I checked the top and both "node" processes were killing the machine. Any reason why?  😎😅🤔
